### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Neuro-Genesis67**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Pull Request Tutorial
 
 For educational purposes, this is a chunk of text that needs to be formatted
-just a little better (and have its typos fixed).
+just a little bettjklkjljkjkljkljlkjkled).
```
Comment (greystate) [inline README.md]: This is some serious volapyke... please use better words :)

**Review by greystate — CHANGES_REQUESTED** [review]
Not good enough 👎

File: README.md (hunk 1 above)
Comment (Neuro-Genesis67) [reply README.md]: Ooops my bad.

**Review by greystate — APPROVED** [review]
Much better! 👍

**Pull request merged**